test/benchmarks/Benchmarks.Trace/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
VS keeps trying to add this file on my machine and I keep deleting it 😅 
@kevingosse are we ok adding this?

Reply:
I had @kevingosse remove this and tell VS to stop generating it through the csproj. We should no longer have these `launchSettings.json` popping up now after rebasing off `master`

Reply:
thanks!